.github/ISSUE_TEMPLATE/feature-request.md

```diff
@@ -0,0 +1,33 @@
+---
+name: Feature Request
+about: Suggest an idea for this project
+title: ''
+labels: enhancement
+assignees: ''
+
+---
+
+------------------------------- delete below -------------------------------
+
+Thank you for submitting a feature request! A few things:
+
+- Please check that a similar feature request has not already been submitted. Adding support to a currently existing feature request by commenting will increase the visibility and importance of your request beyond submitting a duplicate request. 
+
+- If this is a large feature request, please submit it to user voice.
+https://feedback.azure.com/forums/321918-azure-iot
+
+As an open source project, we always welcome PRs for new features. So if you know what needs to be done, feel free to create the change and send it our way.
```

Comment:
We should be explicit.  Customers should create an issue describing the proposed solution before submitting a PR.  The last thing we want to do is reject a community PR :(

Reply:
How would a customer determine whether "this is a large feature request" or not?  Could we give guidance, like "if you know this request requires service changes" then point them to the feedback.azure?

Reply:
Actually, customers may not even know what features require service change (e.g. Java SDK missing X.509 CA).  The guidance should be: if this feature requires implementation in multiple SDKs, consider using user voice.